components/RadioPlayer.vue

```diff
@@ -7,6 +7,14 @@
 
       <div class="now-playing-details">
         <div class="radio-controls">
+          <div v-if="showAlbumArt && np.now_playing.song.art" class="now-playing-art">
+            <a class="cursor-pointer programimage" rel="playerimg" @click.stop="streamModal = !streamModal">
+              <div v-if="show_check == true" class="onair">On air</div>
+              <img class="progimg" :src="show_art_url" :alt="'album_art_alt'">
+            </a>
+            <Modal :media="show_art_url" :title="show_title" :description="show_subtitle" :visibility="streamModal" @close="closeModal" />
```

Comment:
What happens on the `@close` action?

Reply:
@close is the built in method for modals (very clever), it triggers a function in the Modal component

Reply:
Thanks for the teachment